### DISCUSSION TITLE
2016/04/10/beacon-near-plainfield-illinois

### DISCUSSION THREAD

**Discussion by giscus** [jengalas/thesurveystation-jekyll#12] [Announcements]
# 2016/04/10/beacon-near-plainfield-illinois

http://localhost:4000/2016/04/10/beacon-near-plainfield-illinois

**Comment by jengalas** [comment]
Posted by Dave DuPree
February 3, 2017 at 10:24 pm
_Imported from WordPress_

The tower in Plainfield isn’t just any old beacon tower. It’s one of the very early towers erected by the Air Mail Service of the U. S. Post Office Department. These were essentially windmill towers, modified and built by the Baker Manufacturing Company of Evansville, WI. They were originally designed in 1922-23 to support an 18″ rotating beacon. However, by 1925 the 18″ beacons had been deemed inadequate and were replaced with larger and heavier 24″ beacons. In 1927-28, after the Dept. of Commerce assumed responsibility for the federal airways, a beefier, derrick type tower was sought to support a 24″ beacon, two course lights and additional equipment (and to provide a larger, safer work platform). From that point forward, all federal airway beacon towers were evidently supplied solely by the International Derrick & Equipment Co. (IDECO), a manufacturer of steel oil derricks.

Those early Baker towers quickly became symbols not only of the U. S. Air Mail, but of aviation progress in general. Photos of the towers, often with a mail plane overhead, appeared in newspapers all across the country in the 1920s. The towers lined the lighted portions of the first transcontinental airway, and continued in service long after the Dept. of Commerce took over. A few of these old towers were still being used by the FAA to support antennas as late as the 1970s. Then they disappeared. After more than a decade of searching I have found only two survivors. The other, reduced in height by a few feet, serves as an airport beacon tower in Kansas.

The tower in Plainfield is an aviation treasure. Thank you for posting the photos.
